truffleruby/src/main/ruby/core/rbconfig.rb

```diff
@@ -148,6 +144,11 @@ def RbConfig.expand(val, config = CONFIG)
     val.replace(newval) unless newval == val
     val
   end
+
+  CONFIG.each_value do |val|
+    RbConfig::expand(val)
+  end
```

Comment:
It could be in stdlib, but RubyGems requires it at startup so it ends up the same if RubyGems is available.